grape/create.py

```diff
@@ -118,44 +116,68 @@ def create_container_init(conf: dict, waitval: float):
     faster than just doing a simple wait.
 
     Args:
-        conf - the configuration
-        wait - the container create wait time
+        conf: The configuration.
+        waitval: The container create wait time in seconds.
     '''
+    time.sleep(1)  # arbitrary short wait
```

Comment:
Why's this here?

Reply:
To allow the new grape docker containers to start in docker. I definitely need to update the comment.
This was put in place to avoid the a `docker.errors.NotFound` exception in rare cases.

Reply:
Added this comment:
```
    # This is a heuristic that does a short wait to give docker
    # sufficient time to start to define the new containers before we
    # start to query them.
    #
    # In particular, this significantly reduces the chance
    # that the docker.errors.NotFound exception will be
    # raised.
    #
    # One second is probably overkill.
    time.sleep(1)
```